src/index/ivf/ivf_config.h

```diff
@@ -75,6 +76,11 @@ class ScannConfig : public IvfFlatConfig {
 
     inline Status
     CheckAndAdjustForSearch(std::string* err_msg) override {
+        if (!faiss::support_pq_fast_scan) {
+            *err_msg = "scann index can not be used on current cpu, need at least avx2 instruction set";
```

Comment:
"the current index is not supported on the current CPU model. AVX2 support is needed for x86."

Reply:
This check is for one specific index so we can directly point out the index name here, and `AVX2 support is needed for x86` is good indeed